File: Chapitre3/exercice3.1.py
import sys

def main():
    sum = 0
    if len(sys.argv) != 3:
        print("Pas d'argument correct", file=sys.stderr)
        sys.exit(1)
    else:
        for j in range(len(sys.argv) - 1):
            sum += int(sys.argv[j + 1])
        print(f"La somme est: {sum}")

if __name__ == "__main__":
    main()

File: Chapitre3/exercice3.1.c
//prendre 2 arguments en ligne de commande
#include <stdlib.h>
#include <stdio.h>
#include <errno.h>

int main(int argc, char *argv[])
{
    int sum=0;
    if(argc != 3)
    {
        perror("Pas d'argument correct");
        exit(1);
    }
    else
    {
        for(int j=0 ; j<argc-1; j++ )
        {
            sum += atoi(argv[j+1]);
        }
        printf("La somme est: %d\n", sum);
    }
    return 0;
}